### PR TITLE
Bug 1437591 - Add support for x86_64 GVE builds

### PR DESCRIPTION
**Pull request by staktrace**
I just cobbled this together quickly as a minimum viable thing for what I needed. Happy to clean it up as you see fit. I only saw the PR in #565 after I wrote and was kind of surprised at how similar they were, but that one seems more complicated. Also the problem with using `mozinfo.processor` as the default is that it seems to be based on the host machine which we don't really care about so much for Android. My PR keeps the `arch` separate from `processor`.

### CODE REVIEW

File: mozregression/cli.py [hunk 1]
```diff
@@ -550,7 +557,15 @@ def validate(self):
 
         user_defined_bits = options.bits is not None
         options.bits = parse_bits(options.bits or mozinfo.bits)
-        fetch_config = create_config(options.app, mozinfo.os, options.bits, mozinfo.processor)
+        if options.arch is not None:
+            self.logger.info("app is %s" % options.app)
+            if options.app != "gve":
+                self.logger.info("arch option being ignored for non-GVE app.")
```
Comment (wlach) [inline mozregression/cli.py:563]: Let's make this a warning

```suggestion
                self.logger.warning("--arch ignored for non-GVE app.")
```

File: mozregression/cli.py [hunk 2]
```diff
@@ -550,7 +557,15 @@ def validate(self):
 
         user_defined_bits = options.bits is not None
         options.bits = parse_bits(options.bits or mozinfo.bits)
-        fetch_config = create_config(options.app, mozinfo.os, options.bits, mozinfo.processor)
+        if options.arch is not None:
+            self.logger.info("app is %s" % options.app)
```
Comment (wlach) [inline mozregression/cli.py:561]: This looks like a debugging statement that snuck in

```suggestion
```

**Comment by wlach** [comment]
It would be nice to expose this in the GUI as well -- many mozregression users are using that, not asking you to do that here but perhaps we can file a bug to followup

**Comment by staktrace** [comment]
Patch updated to address nits. I also added a second patch which updates the GUI. Tested by manually running it and exercising a bunch of the codepaths, not sure if there's a better way to do that.

Ready for re-review.

**Pull request merged**